src/Sylius/Bundle/ReviewBundle/Doctrine/ORM/Subscriber/LoadMetadataSubscriber.php

```diff
@@ -106,11 +106,13 @@ private function createReviewerMapping($reviewerEntity, ClassMetadata $reviewerE
         return [
             'fieldName' => 'author',
             'targetEntity' => $reviewerEntity,
-            'joinColumn' => [
-                'name' => 'author_id',
-                'referencedColumnName' => $reviewerEntityMetadata->fieldMappings['id']['columnName'],
-                'nullable' => false,
-                'onDelete' => 'CASCADE',
+            'joinColumns' => [
```

Comment:
We used to put both arrays openings in the same line (check [LoadMetadataSubscriber in attribute bundle](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AttributeBundle/Doctrine/ORM/Subscriber/LoadMetadataSubscriber.php#L85-L90)).